effekt/shared/src/main/scala/effekt/core/DirectStyleMutableState.scala

```diff
@@ -24,7 +24,7 @@ object DirectStyleMutableState extends Phase[CoreTransformed, CoreTransformed] {
 
   override def run(input: CoreTransformed)(using Context): Option[CoreTransformed] = input match {
     case CoreTransformed(source, tree, mod, core) =>
-      val direct = directStyle.rewrite(core)
+      val direct = Context.timed(phaseName, source.name)(directStyle.rewrite(core))
```

Comment:
```suggestion
      val direct = Context.timed(phaseName, source.name) { directStyle.rewrite(core) }
```
Just to be consistent